common/templates/searchInput.html

```diff
@@ -1,8 +1,15 @@
 <div class="chaise-search-box chaise-input-group" ng-class="{'disabled-element': disabled}">
     <div class="chaise-input-control has-feedback">
         <input type="text" ng-model="searchTerm" ng-class="inputClass" ng-disabled="disabled" on-enter="enterPressed()" ng-change="inputChanged()" ng-attr-autofocus="{{focus === true}}">
-        <span ng-hide="searchTerm" class="chaise-input-placeholder" ng-click="changeFocus()" ng-if="placeholder.isHTML === true" ng-bind-html="'Search ' + placeholder.value"></span>
-        <span ng-hide="searchTerm" class="chaise-input-placeholder" ng-click="changeFocus()" ng-if="placeholder.isHTML === false" ng-bind="'Search '  + placeholder.value"></span>
+        <span ng-hide="searchTerm" class="chaise-input-placeholder" ng-click="changeFocus()" ng-if="!isArray(searchColumns)" ng-bind="'Search' + (searchColumns === false ? ' all columns': '')"></span>
```

Comment:
Is this suggesting that `searchColumns` could be a single column name? Just curious because the conditional here could be moved down below and conditionally add " all columns" OR  ng-repeat through `searchColumns`.

Reply:
`searchColumns` could be `false` (if it's used in entity mode and the `search-box` annotation is missing).